src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -251,6 +254,11 @@ public void onNewIntent() {
             reloadList();
             updateLocationButton();
         }
+
+        // hack (alternatives welcome): we position the AddReactionView offscreen to "-666dp"
+        // to get the real width/height before it is shown;
+        // after first layout we can hide it so that eg. screen readers are not confused
+        addReactionView.hide();
```

Comment:
doesn't `View.INVISIBLE` works like that? the docs[*] say:
> This view is invisible, but it still takes up space for layout purposes.

so I assume the layout size is calculated, this would avoid that weird hack

`*` https://developer.android.com/reference/android/view/View.html#INVISIBLE

Reply:
thanks for the hint, i'll check that out!